src/components/carousel/index.jsx

```diff
@@ -108,6 +136,36 @@ const Carousel = ({
 		<DefaultPreviousButton id={id} onClick={() => previousSlide()} />
 	);
 
+	const resolvedFullScreenShowButton = fullScreenShowButton ? (
+		cloneElement(fullScreenShowButton, {
+			onClick: toggleFullScreen,
+			className: `${COMPONENT_CLASS_NAME}__full-screen-toggle ${fullScreenShowButton.props?.className}`,
+		})
+	) : (
+		<button
+			onClick={toggleFullScreen}
+			type="button"
+			className={`${COMPONENT_CLASS_NAME}__full-screen-toggle`}
+		>
+			Full Screen
+		</button>
```

Comment:
Wondering if we shouldn't mimic the prev/next buttons here.  If we use the standard Button component then it can be styled in a consistent manner for each state.  Currently, it would be styled differently for this specific button.

Thoughts?

Reply:
updated @nschubach